starter-code/src/viking.js

```diff
@@ -1,11 +1,111 @@
 // Soldier
-function Soldier () {}
+function Soldier (health, strength) {
```

Comment:
Aquí veo un problema que se repite en las distintas clases que has creado en el ejercicio. Abres las llaves al inicio de la clase y las cierras al final, incluyendo los prototypes dentro de ellas, esto en Javascript no se hace así, en otros lenguajes como Java si que se meten los métodos dentro de la clase. Aquí lo que debemos hacer es algo así:

function Soldier (health, strength) {
  this.health = health;
  this.strength = strength;
}

Soldier.prototype.attack = function(){
  return this.strength;
}